metrics/report/report_dockerfile/scaling.R

```diff
@@ -92,7 +90,7 @@ for (currentdir in resultdirs) {
 					fudata=rbind(fudata,frow)
 				}
 			}
-			colnames(fudata)=c("node", "noschedule", "cpu_idle", "mem_free", "pod", "testname")
+			colnames(fudata)=c("node", "noschedule", "cpu_idle", "mem_free", "pod", "testname", "render_point")
```

Comment:
I think this "render_point" addition here is where the array index error comes from - as there is not matching extra cbind into the fudata rows above afaict. I found I could not build a plot of a multi-node run either. I removed the addition here, and now it builds/displays.